### DISCUSSION TITLE
iptables für die FritzBox 7362SL

### DISCUSSION THREAD

**Discussion by HTMofDreams** [Freetz-NG/freetz-ng#573] [Unerfüllte Wünsche] [answered]
Ich habe mir die ( FritzBox 7362SL ) Kernelmodule gebaut für die Firmware 7.13 mit freetz-ng:

`root@freetz-flur:/lib/net/ipv4/netfilter$ ls -l
-rw-r--r--    1 root     root         23260 Jun 23 06:33 ip_tables.ko
-rw-r--r--    1 root     root          6328 Jun 23 06:33 ipt_MASQUERADE.ko
-rw-r--r--    1 root     root          6360 Jun 23 06:33 ipt_REJECT.ko
-rw-r--r--    1 root     root         12796 Jun 23 06:33 ipt_ULOG.ko
-rw-r--r--    1 root     root          5276 Jun 23 06:33 iptable_filter.ko
-rw-r--r--    1 root     root          5360 Jun 23 06:33 iptable_mangle.ko
-rw-r--r--    1 root     root          7540 Jun 23 06:33 iptable_nat.ko
-rw-r--r--    1 root     root         22552 Jun 23 06:33 nf_conntrack_ipv4.ko
-rw-r--r--    1 root     root          4272 Jun 23 06:33 nf_defrag_ipv4.ko
-rw-r--r--    1 root     root          9908 Jun 23 06:33 nf_nat_ipv4.ko
root@freetz-flur:/lib/net/ipv4/netfilter$`

Nun ist es so wenn ich dieses Modul nf_defrag_ipv4.ko per insmod laden möchte, schmiert die Box einfach so ab ohne etwas in der Konsole zu sehen. Auch per syslog kann ich nicht sehen, wo ran das liegen mag.
Benutze den eigentlichen AVM Kernel ( also habe kein Kernel Replace ausgewält ). Kann mir da jemand helfen? Oder geht es zum verrecken nicht, iptables und NAT auf der Box zu verwenden?

**Comment by HTMofDreams** [comment]
oder zu mindest mit der alten firmware ...

**Comment by fda77** [comment, marked as answer]
Das steht in der FAQ

**Comment by HTMofDreams** [comment]
Hab ich dann jetzt gelesen. Das heißt dann eher 'Nein'
`Da AVM den Quellcode des PacketAccelerator veröffentlicht hat, könnte das Problem von jemandem mit viel Zeit und Wissen behoben werden.`
Und so viel Wissen habe ich dann leider nicht ... Auch wenns jetzt nicht hier hin gehört aber gibts eine solche Lösung mit einem anderen DSL Router?

**Comment by fda77** [comment]
Jo, das verbuggte Fritzos runterwerfen und was funktionierendes installieren. Mit Openwrt gibts komplette iptables, vlan, multi ssid, routing wie man will , wpa3 etc
![image](https://user-images.githubusercontent.com/471850/175398191-99889923-95f5-4df1-8e30-123650ce20a2.png)

**Comment by HTMofDreams** [comment]
Ich bin dabei ;) lese mich gerade bei ein ;) kriegs noch nicht so richtig hin ... Kernel Image bekomme ich hin aber dann bei scp hänge ich noch ... Klar ip von Rechner ändern und so, kein Thema ...  Aber ich würde ne Flasche Vodka verwetten das dann doch wieder alles nicht geht. Also DECT, brauche ich nicht ( weil geht eh nicht ). Aber diese Repeater Funktion wäre schon wichtig ...

**Comment by HTMofDreams** [comment]
Also nach rüber kopieren des kernel-image blinkt auch die Power-Leuchte. So weit gut ...
Dann IP ändern auf 192.168.1.x auch kein Thema ... Dann sollte man ja eigentlich per SCP drauf kommen und da hänge ich ...

**Comment by fda77** [comment]
Genau frag besser im openwrt forum. Alles was closed source ist wird mit einem aktuellen Kernel nicht gehen, also dect, isdn, pots. Installiert man einfach Asterisk, funktioniert eh besser. Normal geht Wlan, Lan, Dsl
Mehr kann ich dazu nicht sagen

![image](https://user-images.githubusercontent.com/471850/175406948-acc44efd-eddf-4e64-9165-fb52e056381a.png)

**Comment by HTMofDreams** [comment]
hab doch gleich gesagt geht nicht ;) Will ja keiner hören auf mich ;) 
Box bleibt beim booten einfach stehen ... Wie auch bei der AVM/freetz-ng version ... da kannste echt nix machen ...
Ich gehe mal stark davon aus auf einer 7320 bekommste unter freetz-ng auch die iptables mit nat komplett hin. aber nicht auf einer 7362SL ...

**Comment by HTMofDreams** [comment]
Gibt es denn noch etwas wo mit die Box vernünftig rennt? mit eben den besagten iptables?

**Comment by fda77** [comment]
ka, vielleicht hilft das weiter: https://www.google.com/
